.github/workflows/ci_tests_dev.yaml

```diff
@@ -98,8 +98,13 @@ jobs:
       # Pull baseline image data from dvc remote (DAGsHub)
       - name: Pull baseline image data from dvc remote
         run: |
+          ORIGINAL_BRANCH=$(git branch --show-current)
+          # Pull down GMT 6.4 baseline images from
+          # https://github.com/GenericMappingTools/pygmt/pull/1791
+          git checkout update-baseline-images-for-gmt-dev
           dvc pull
           ls -lhR pygmt/tests/baseline/
+          git checkout ${ORIGINAL_BRANCH}
```

Comment:
I'm wondering if we should have a `gmt-6.4` or a more general `gmt-dev` branch.

So changes in https://github.com/GenericMappingTools/pygmt/pull/1791 can be merged into the `gmt-dev` branch and the `ci_tests_dev` workflow can test the `gmt-dev` branch instead.

Of course, we need to keep the `gmt-dev` branch up-to-date with the `main` branch, either manually or via workflow.

Reply:
> I'm wondering if we should have a `gmt-6.4` or a more general `gmt-dev` branch.

A `gmt-6.4` branch sounds good, best not to have a very long-running branch like `gmt-dev` to avoid scrolling down so much in a PR.

> So changes in #1791 can be merged into the `gmt-dev` branch and the `ci_tests_dev` workflow can test the `gmt-dev` branch instead.
> 
> Of course, we need to keep the `gmt-dev` branch up-to-date with the `main` branch, either manually or via workflow.

Let's start with manual for now, and if things break very often, we can have a scheduled cron job to update the `gmt-6.4` branch every week or so.

Reply:
I've created the `gmt-6.4` branch (https://github.com/GenericMappingTools/pygmt/tree/gmt-6.4).